src/inc/public/AppxPackaging.hpp

```diff
@@ -1733,6 +1746,14 @@ MSIX_API HRESULT STDMETHODCALLTYPE PackPackage(
 
 #endif // MSIX_PACK
 
+MSIX_API HRESULT STDMETHODCALLTYPE PackBundle(
+    MSIX_BUNDLE_OPTIONS bundleOptions,
+    char* directoryPath,
+    char* outputBundle,
+    char* mappingFile,

```

Comment:
directoryPath and mappingFile are in inputs for what is going to get bundled, right? I think this might be a little confusing and you might end up with if directoryPath != null && mappingFile != null then bad input.

I think it will look cleaner for callers if there's PackBundle that takes the directory to bundle and PackBundleFromMappingFile. Internally both will call into the same logic

Reply:
I added checks for when directoryPath != null && mappingFile != null within this method. Also added checks for other possible scenarios when directoryPath != null && mappingFile != null.